### PR TITLE
Attribution Changes

### PR DESCRIPTION
**Pull request by swpoole**
Most of these are the  changes required for deprecating the Fortran API.

### CODE REVIEW

File: utils/coverpage.tex [hunk 1]
```diff
@@ -0,0 +1,118 @@
+\thispagestyle{empty}
```
Comment (nspark) [inline utils/coverpage.tex]: Why are there `content/coverpage.tex` _and_ `utils/coverpage.tex` files?

Comment (swpoole) [reply utils/coverpage.tex]: Obviously I uploaded too many files. I can delete them and add them to another pull request or you can delete them.

Comment (swpoole) [reply utils/coverpage.tex]: The coverage and utils should be a different pull. I also did some trivial mods to programming model.

Comment (swpoole) [reply utils/coverpage.tex]: coverpage

**Comment by nspark** [comment]
This PR is being moved to https://github.com/RaymondMichael/specification/pull/3

**Pull request closed without merging**